internal/pk/pk.go

```diff
@@ -0,0 +1,20 @@
+package pk
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func Convert(table *schema.Table, resource []any) string {
```

Comment:
Could have a better name (with a godoc) maybe `Stringify`? Also why not get `schema.Resource` as the single param, and use `ToDestinationResource`?

```suggestion
func Stringify(r *schema.Resource) string {
```

Reply:
we are working with the data from destination side, so we have table + `[][]any`